src/main/java/fr/sncf/osrd/speedcontroller/generators/DichotomyControllerGenerator.java

```diff
@@ -89,6 +89,8 @@ protected double evalRunTime(Simulation sim, TrainSchedule schedule, Set<SpeedCo
             else
                 higherBound = nextValue;
             nextValue = (lowerBound + higherBound) / 2;
+            saveGraph(nextSpeedControllers, sim, schedule,
+                    "C:\\Users\\9706809E\\OneDrive - SNCF\\Documents\\Physique\\Affichage_courbes\\graph" + i + ".csv");
```

Comment:
I think this was use for debug purpose and shouldn't be merge.

Reply:
Yes you're right, i'll erase it.